src/content/app/tools/blast/utils/tests/blastFormValidator.test.ts

```diff
@@ -0,0 +1,66 @@
+/**
+ * See the NOTICE file distributed with this work for additional information
+ * regarding copyright ownership.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {
+  isBlastFormValid,
+  BLAST_SPECIES_LIMIT,
+  BLAST_SEQUENCE_LIMIT
+} from '../blastFormValidator';
+
+const createFakeSpecies = (times: number) => {
+  return Array(times).fill('');
+};
+
+const createFakeSequences = (times: number) => {
+  return Array(times).fill({
+    value: 'ACTG'
+  });
+};
+
+describe('isBlastFormValid', () => {
+  it('fails validation if we have number of species more than the limit', () => {
```

Comment:
`if we have number of species more than` -> `if the number of species is greater than` (less personal, no "we")

Same for other test descriptions below

Reply:
That's done